server/src/main/java/org/elasticsearch/action/ingest/DeletePipelineResponse.java

```diff
@@ -0,0 +1,62 @@
+/*
+ * Licensed to Elasticsearch under one or more contributor
+ * license agreements. See the NOTICE file distributed with
+ * this work for additional information regarding copyright
+ * ownership. Elasticsearch licenses this file to you under
+ * the Apache License, Version 2.0 (the "License"); you may
+ * not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+
+package org.elasticsearch.action.ingest;
+
+import org.elasticsearch.action.support.master.AcknowledgedResponse;
+import org.elasticsearch.common.io.stream.StreamInput;
+import org.elasticsearch.common.io.stream.StreamOutput;
+import org.elasticsearch.common.xcontent.ConstructingObjectParser;
+import org.elasticsearch.common.xcontent.ToXContentObject;
+import org.elasticsearch.common.xcontent.XContentParser;
+
+import java.io.IOException;
+
+public class DeletePipelineResponse extends AcknowledgedResponse implements ToXContentObject {
```

Comment:
can you remind why we need this? I saw that we introduced PutPipelineResponse in the put pipeline PR as well, but I wonder why we cannot use the existing WritePipelineResponse.

Reply:
You are right. I missed this common response object. I will add the changes here and remove the two files. However, it would look a bit odd for high-level-client users that `DeletePipelineRequest` has a `WritePipelineResponse` even though functionally they are the same.

Reply:
I think that's ok, plus they will get what the transport client used to return which is good as long as the response doesn't expose weird internal stuff.